src/basic/forward.h

```diff
@@ -6,6 +6,7 @@
 #include <errno.h>              /* IWYU pragma: export */
 #include <inttypes.h>           /* IWYU pragma: export */
 #include <limits.h>             /* IWYU pragma: export */
+#include <paths.h>              /* IWYU pragma: export */
```

Comment:
where is this needed?

Reply:
This defines _PATH_BSHELL, which is used several places in PID1, executor, run.
Currently, no one includes the header explicitly, but indirectly through e.g utmpx.h, libmount.h, mntent.h, and so on.

Reply:
Please note this in the commit message, as it's unclear